ts/features/bonus/cgn/screens/merchants/CgnMerchantDetailScreen.tsx

```diff
@@ -215,14 +166,34 @@ const CgnMerchantDetailScreen = () => {
                 value={merchantDetail.value.description}
               />
               <VSpacer size={24} />
-              {renderMerchantAddressesList(
-                merchantDetail.value.addresses,
-                merchantDetail.value.allNationalAddresses
-              )}
+              {showAddresses || showGotToWebsite ? (
```

Comment:
I suggest to replace the ternary operator (? :) with the logical AND (&&) operator in our conditional renderings for better readability and simplicity.